fsh-tank/input/fsh/CervicalCancerScreeningActivity.fsh

```diff
@@ -21,10 +21,9 @@ Usage: #definition
 * intent = $RINTENT#proposal "Proposal"
 * priority = $RPRIOR#routine "Routine"
 * dynamicValue[+].path = "orderDetail" // this is the ServiceRequest.orderDetail element
-* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql "CQL"
+* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql-identifier "CQL"
 * dynamicValue[=].expression.expression = "ServiceRequestOrderDetail"
-* dynamicValue[=].expression.reference = "Library/ScreeningLibrary|1.0"
```

Comment:
The Jira ticket says to remove "the reference attributes in the action expression fields in the ActivityDefinition and PlanDefinition resources", however, these ones are in the dynamic value field. Do you know whether or not these should be removed? Whatever we decide, we should keep it consistent between the two dynamic value entries (here one library is removed, but one remains).

Reply:
yeah I think they all should be removed dynamic or not.  They all should pull from the library element.  I think the only exception would be if there was more than 1 library in the list, in which case they'd have to specify.  But I think that's only an issue with the flat plandefs.